src/analysis/special_functions/exponential.lean

```diff
@@ -24,17 +24,23 @@ We prove most result for an arbitrary field `𝕂`, and then specialize to `𝕂
   `1 : 𝔸 →L[𝕂] 𝔸` at zero, as long as it converges on a neighborhood of zero
   (see also `has_strict_deriv_at_exp_zero_of_radius_pos` for the case `𝔸 = 𝕂`)
 - `has_strict_fderiv_at_exp_of_lt_radius` : if `𝕂` has characteristic zero and `𝔸` is commutative,
-  then given a point `x` in the disk of convergence, `exp 𝕂` as strict Fréchet-derivative
+  then given a point `x` in the disk of convergence, `exp 𝕂` has strict Fréchet-derivative
   `exp 𝕂 x • 1 : 𝔸 →L[𝕂] 𝔸` at x (see also `has_strict_deriv_at_exp_of_lt_radius` for the case
   `𝔸 = 𝕂`)
+- `has_strict_fderiv_at_exp_smul_const_of_mem_ball`: `u ↦ exp 𝕂 (u • x)`, where `x` belongs to a
+  ring that may not be commutative, has strict Fréchet-derivative
+  `exp 𝕂 (t • x) • (1 : 𝔸 →L[𝕂] 𝔸).smul_right x` at `t` if `t • x` is in the radius of convergence.
```

Comment:
That's a bit hard to understand IMHO, what do you think of this?
```suggestion
- `has_strict_fderiv_at_exp_smul_const_of_mem_ball`: even when `𝔸` is non-commutative,
  if we have an intermediate algebra `𝔹` which is commutative, then
  the function `(u : 𝔹) ↦ exp 𝕂 (u • x)`, still has strict Fréchet-derivative
  `exp 𝕂 (t • x) • (1 : 𝔹 →L[𝕂] 𝔹).smul_right x` at `t` if `t • x` is in the radius of convergence.
```

(I exchanged the name `𝔸` and `𝔹` since that fits better with the rest of the docstring, maybe we should also change them in the lemmas for consistency?)

Reply:
I've renamed the variables to `𝕂 𝕊 𝔸`; I thought having a tower of algebras with `B` before `A` would be confusing. I also considered `𝕂 𝕃 𝔸` but as I understand it `L` usually refers to fields.